modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -64,7 +64,16 @@
 	<td>//div[@class='dropdown-menu show']</td>
 	<td></td>
 </tr>
-
+<tr>
+	<td>LANGUAGE_TRANSLATION_IN_DROPDOWN</td>
+	<td>//div[@class='dropdown-menu show']//*[contains(@class,'lexicon-icon-${key_languageTranslationOption}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS</td>
+	<td>//button[contains(@class,'dropdown-item')]//div[contains(@class,'autofit-section')and text()='Manage Translations']</td>
+	<td></td>
```

Comment:
The path could be called as : MANAGE_TRANSLATIONS_BUTTON. Cause you are creating a path to open the manage translations component, clicking on the button. This way it would be more clear to other QAs/etc what it's the path context.